scripts/imgtool/dumpinfo.py

```diff
@@ -251,7 +251,7 @@ def dump_imginfo(imgfile, outfile=None, silent=False):
     ###############################################################################
 
     if silent:
-        sys.exit(0)
+        return
```

Comment:
for backward compatibility, you can skip printing message `dumpinfo has run successfully` in main.py:dumpinfo method when silent option is used (just add `if not silent:`)